content/documents/building-wishlist-using-product-collections.mdx

```diff
@@ -0,0 +1,173 @@
+---
+type:           tutorial
+title:          Building a Wishlist Feature with Product Collections
+description:    A product wishlist feature is one of the many permutations of the OrderCloud Product Collections resource.
+author:         dsteinmetz
+publishDate:    2022-10-11
+updatedDate:    2022-10-11
+tags:           ["Community", "Product Collections", "Wishlist", "Favorites", "Save For Later"]
+---
+
+## Prerequisites
+To make use of the [Product Collections](/knowledge-base/product-collections) resource, we need:
+- An active buyer user
+- Fully orderable products visible to the buyer user
+
+## Assumptions
+- For this tutorial, we are using the URL for the sandbox environment in the region US-West.
+- We have a product with an ID of `enterprise-product-id` that is orderable and visible to our buyer user.
+- We end up with a product that had been added to our Wishlist that we want to remove, with an ID of `basic-product-id`.
```

Comment:
What if you remove this assumption and then in the tutorial simply remove the product you added in the tutorial? You could illustrate the list capabilities before the delete so that you can still show the listing piece

Reply:
That is probably more clear.  I can make this change, thanks for the suggestion.

Reply:
Updated